### PR TITLE
KIALI-358 Adds a MessageCenter component for showing alert notifications

### PR DESCRIPTION
**Pull request by josejulio**
This PR covers the stateless component MessageCenter and would like feedback regarding the look and how it should behave. Next steps will be to couple with redux to be able to use it on any other component trough a redux action.

Patternfly [suggests](http://www.patternfly.org/pattern-library/communication/notification-drawer/) to use the NotificationDrawer along with the ToastNotification for displaying recent notifications upon login.

peek 2018-04-23 10-14

~~I still need to squash my commits (aka clean my mess) :)~~

### CODE REVIEW

**Comment by josejulio** [comment]
screenshot from 2018-04-18 23-40-28

WIP: Also adding the NotificationDrawer, separating it into some subcomponents.

**Comment by josejulio** [comment]
peek 2018-04-19 18-44

**Comment by mtho11** [comment]
Be sure to squash.

**Comment by josejulio** [comment]
Squashed, removed some commented code and filled the `index.ts` for easier importing in other modules.

File: src/components/MessageCenter/MessageCenter.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import NotificationDrawer from './NotificationDrawer';
+import NotificationList from './NotificationList';
+
+import { NotificationMessage, NotificationGroup, MessageCenterPropsType } from './Types';
+
+type StateType = {};
+
+export default class MessageCenter extends React.Component<MessageCenterPropsType, StateType> {
+  getNotificationMessages = (groups: NotificationGroup[]) => {
```
Comment (vnugent) [inline src/components/MessageCenter/MessageCenter.tsx]: could you add a comment or unit test explaining what this function does?

**Comment by jotak** [comment]
(you have a compile error in MessageCenter.test.tsx )

**Comment by josejulio** [comment]
> (you have a compile error in MessageCenter.test.tsx according to travis)

damn travis!

**Pull request merged**